apps/server/src/modules/provisioning/strategy/schulconnex/schulconnex.strategy.ts

```diff
@@ -96,6 +101,11 @@ export abstract class SchulconnexProvisioningStrategy extends ProvisioningStrate
 
 			await Promise.all(groupProvisioningPromises);
 		}
+
+		const endTime = performance.now();
+		this.logger.warning(
+			new GroupProvisioningInfoLoggable(data.externalUser, data.externalGroups ?? [], endTime - startTime)
+		);
```

Comment:
At the moment we only print log messages at the log level error on production.
So put it's to the log level error on the moment that we cann see it at production

Reply:
That logging go from error down to warning on production is not posible by data protection.
One example for to manny informations are logged on warning is https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/group/uc/group.uc.ts#L48